### PR TITLE
Add a new (repeat str n times) problem (L-I)

### PR DESCRIPTION
**Pull request by ibr5500**
## In this [PR] I have:

> - Added a new problem (repeat a string number of times).
> - Created a README file and added some details.
> - Created a test file for testing.
> - Installed Mocha library for testing proposes.

### CODE REVIEW

File: L-I/0006 Repeat a String (L-I)/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# 0006 Repate a String Num times
+
+repeat  a given string `str` for `num` of times, and return empty in case the num is zero or negative number. 
+
+#### Note: you are not allowd to use `.repeat()` method.
+
+## For testing:
```
Comment (devvsakib) [inline L-I/0006 Repeat a String (L-I)/README.md]: title is Test Cases
provide some test cases

File: L-I/0006 Repeat a String (L-I)/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+```
+const repeatStringNumTimes = (str, num) => {
+  if (num < 1) {
+    return "";
+  }
+  // here I used recursion technique to solve the problem
+  return str + repeatStringNumTimes(str, num - 1);
+};
+```
+
```
Comment (devvsakib) [inline L-I/0006 Repeat a String (L-I)/README.md:25]: Here you have to add How it woks title and explain how your code works.

File: L-I/0006 Repeat a String (L-I)/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+[click](https://www.khanacademy.org/computing/computer-science/algorithms/recursive-algorithms/a/recursion) To lern more about recursion method 
+
```
Comment (devvsakib) [inline L-I/0006 Repeat a String (L-I)/README.md:44]: please follow the guidelines on how you can make a perfect readme file [link](https://github.com/devvsakib/javascript-problem-solving/blob/main/L-B/0002%20ContainString%20(%20L-B%20))/README.md)

**Review by devvsakib — CHANGES_REQUESTED** [review]

**Review by devvsakib — APPROVED** [review]
Awesome

**Comment by ibr5500** [comment]
> Awesome

Thank you @devvsakib

**Pull request merged**